test/unit/lib/utils/queryMapper.js

```diff
@@ -49,86 +49,179 @@ describe('queryMapper', () => {
     });
   });
 
-  describe('getResultsHeading', () => {
-    const location = 'ls1';
-    it('should return undefined when empty query', () => {
+  describe('getResultsInfo', () => {
+    it('should return false when empty query', () => {
       const query = {};
-      const output = queryMapper.getResultsHeading(query, location);
-      expect(output).to.equal(undefined);
+      const location = 'ls1';
+      const output = queryMapper.getResultsInfo(query, location);
+      expect(output).to.eql({
+        correctResultsParams: false,
+        resultsExplanation: false,
+        resultsHeading: false,
+        resultsOnwardsJourneyPartial: false,
+      });
     });
 
-    it('should return undefined when empty location', () => {
+    it('should return false when empty location', () => {
       const emptyLocation = undefined;
       const query = {
         origin: constants.serviceChoices.symptoms,
         type: constants.serviceTypes.professional,
       };
-      const output = queryMapper.getResultsHeading(query, emptyLocation);
-      expect(output).to.equal(undefined);
+      const output = queryMapper.getResultsInfo(query, emptyLocation);
+      expect(output).to.eql({
+        correctResultsParams: false,
+        resultsExplanation: false,
+        resultsHeading: false,
+        resultsOnwardsJourneyPartial: false,
+      });
     });
 
     it('should return a sexual professionals message when the query is related to professionals and origin is any', () => {
```

Comment:
The title of the test doesn't match what is being tested. If the origin could be any, perhaps have a test for each of the different origins, this should avoid confusion in the future and catch changes that affect the case where the origin isn't what is specifically being tested here.